src/App.jsx

```diff
@@ -14,33 +17,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const CONDITION_LENGTH = 'length';
+const CONDITION_ALPHABET = 'alphabet';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function prepareGoods(condition, reversed) {
```

Comment:
as I see you can move this function to another file then import and use the inside component as a helper, util func.

Reply:
Hi! 
I fixed everything, thank you!
Moved my function and constants into separate files